### DISCUSSION TITLE
Adding additional sensors switches to ESP32

### DISCUSSION THREAD

**Discussion by stevenbrown1862** [Yogui79/IntexPureSpa#48] [Ideas]
Hi. Firstly thanks to the creators for a great project that works really well!  What I would like to do to is make use of some of the other unused GPIO pins on the esp32 by adding control of a relay and a dht22... I've done quite a bit with esphome but never done anything with arduino code so have absolutely no idea how I could add this and have it communicate by mqtt.   Anyone clever enough with some spare time to spoonfeed me some lines that I could add to the sketch?

**Comment by Yogui79** [comment]
Hello ,

sorry I haven' see your message earlyer. I'm actualy bussy but I will help you i's very easy

**Reply by stevenbrown1862** [reply]
> 

Thanks.  I'd appreciate any advice that you can give. Whenever you get time.  Thanks again for a fantastic project!
